client/src/components/Signup/Signup.js

```diff
@@ -102,64 +141,75 @@ export default function SignUp() {
                           type="password"
                           id="password"
                           autoComplete="current-password"
+                          InputProps = {{ 
+                            classes: {root: classes.textFieldSelectLabel} 
+                          }}
+                          InputLabelProps = {{ 
+                            classes: {root: classes.textFieldSelectLabel} 
+                          }}
+                          value={formValues.password}
+                          onChange= { handleChange }
+                          error = { formErrors.password }
+                          helperText= { formErrors.password || null}
+                        />
+                      </Grid>
+                      <Grid item xs={12}>
+                        <TextField
+                          required
+                          fullWidth
+                          name="confirmPassword"
+                          label="Confirm Password"
+                          type="password"
+                          id="confirmPassword"
+                          autoComplete="current-password"
+                          InputProps = {{ 
+                            classes: {root: classes.textFieldSelectLabel} 
+                          }}
+                          InputLabelProps = {{ 
+                            classes: {root: classes.textFieldSelectLabel} 
+                          }}
+                          value={formValues.confirmPassword}
+                          onChange = { handleChange }
+                          error = { formErrors.confirmPassword }
+                          helperText= { formErrors.confirmPassword || null}
                         />
                       </Grid>
                       <Grid item xs={12}>
                         <FormControl className={classes.formControl}>
-                          <InputLabel shrink id="demo-simple-select-placeholder-label-label">
-                            Age
+                          <InputLabel classes = {{ root: classes.selectRoot }} shrink id="demo-simple-select-placeholder-label-label">
+                            Select Primary Language
                           </InputLabel>
                           <Select
+                            fullWidth
                             labelId="demo-simple-select-placeholder-label-label"
```

Comment:
add name="language" in order for selection to work properly

Reply:
Great find! Fixed it.